Combat/Source/Points.cpp

```diff
@@ -50,9 +50,23 @@ void Points::resetCurrent()
    _currentPoints = _maxPoints;
 }
 
+void Points::setCurrentPoints(const uint newValue)
+{
+  _currentPoints = newValue;
+  if(_currentPoints > _maxPoints)
+  {
+     _currentPoints = _maxPoints;
+  }
+}
+
+uint Points::getCurrentPoints() const
+{
+  return _currentPoints;
+}
+
 const bool operator == (const Points& lhs, const Points& rhs)
```

Comment:
usunąć wszędzie spacje pomiędzy operator a == / < / > itd. Wydaje mi się, że już to Wam pisałem.